File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/PreAuthorisationUnderAssessReason.cs
namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class PreAuthorisationUnderAssessReason : Common.Entities.AuditDetails
    {
        public int PreAuthorisationUnderAssessReasonId { get; set; }
        public int PreAuthId { get; set; }
        public int UnderAssessReasonId { get; set; }
        public string UnderAssessReason { get; set; }
        public string Comments { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/preauth-manager/models/preauthorisation-underassessreason.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';
export class PreAuthorisationUnderAssessReason extends BaseClass
{
    preAuthorisationUnderAssessReasonId: number;
    preAuthId: number;
    underAssessReasonId: number;
    underAssessReason: string;
    comments: string;
}